src/docs/ftw-analytics/how-to-set-up-analytics-for-ftw/index.md

```diff
@@ -1,61 +1,138 @@
 
-FTW supports tracking page views with pluggable analytics handlers.
+FTW comes with built-in support for Google Analytics and supports
+tracking page views with customisable analytics handlers. This article
+explains how to enable Google Analytics and use and create custom
+analytics handlers.
 
-## Configure Google Analytics ID
+## Configure Google Analytics
 
-Google Analytics (GA) is supported by default. To enable GA, set the GA
-tracker ID to the environment variable `REACT_APP_GOOGLE_ANALYTICS_ID`;
+FTW has built-in support for Google Analytics. All you need to do is
+assign your Google Analytics Tracking ID to the environment variable
+`REACT_APP_GOOGLE_ANALYTICS_ID`.
 
-Note that the current script expects a _Universal Analytics property
-only_ tracking ID. If you want to use GA4, you should update the Google
-Analytics scripts first.
+> Google Analytics doesn't work in a hot-loading environment! The
+> analytics script is added server-side. You can test it in your local
+> environment by using the command `yarn run dev-server`.
 
-That's it! You're good to go.
+### Google Analytics 4
 
-> Note: Google Analytics doesn't work in hot-loading environment! <br>
-> GA script is added in server-side setup. You can test it local
-> environment with `yarn run dev-server`.
+Google recently released their new analytics service Google Analytics 4.
+Support for Google Universal Analytics will end on October 1, 2023. New
+versions of FTW provide out-of-the-box support for Google Analytics 4.
 
-## Optional: Use a custom analytics handler
+If you're starting development on a new version of FTW and prefer to use
+Universal Analytics, you should look into how Analytics was implemented
+before
+[this pull request](https://github.com/sharetribe/ftw-daily/pull/1508).
+If you have built your marketplace on top of an older version of FTW and
+you want to start using Google Analytics 4, you'll need to implement the
+changes introduced in
+[this pull request](https://github.com/sharetribe/ftw-daily/pull/1508).
 
-If you want to add a new analytics library, you can do as follows:
+> New versions of FTW will expect a Tracking ID compatible with Google
+> Analytics 4. FTW expects the ID to begin with the "G-" prefix.
 
-### 1. Add the analytics library script
+#### Enhanced measurements
 
-If the analytics library has an external script, add the library script
-tag to the
+It is not recommended to use the Enhanced Measurements feature
+introduced in Google Analytics 4, which is enabled by default. The
+Enhanced Measurements feature injects code into link tags which can
+break in-app navigation in FTW. Therefore, we strongly recommend
+disabling the Enhanced Measurements feature when using Google Analytics
+4 with FTW.
+
+<video>
+    <source src='./turn-off-enhanced-measurements.mp4' type='video/mp4'>
+    <source src='./turn-off-enhanced-measurements.webm' type='video/webm'>
+    <source src='./turn-off-enhanced-measurements.ogv' type='video/ogg'>
+</video>
```

Comment:
Great use of video! 👌

Reply:
Thanks! Not *that* necessary here but I wanted to try it out 😆